plugins/core/IO/qDracoIO/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+option( PLUGIN_IO_QDRACO "Install qDracoIO plugin to read & write DRC files" OFF )
+
+if( PLUGIN_IO_QDRACO )
+	project( QDRACO_IO_PLUGIN )
+
+	set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/modules)  
```

Comment:
It seems the files contained in this directory have not been added to the commit
And `list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/modules)` might be better to not override the `CMAKE_MODULE_PATH`

Reply:
Ah, that was a (failing) attempt to help CMake find the module...